python/lsst/pipe/tasks/configurableActions/_configurableActionStructField.py

```diff
@@ -137,6 +139,10 @@ def __init__(self, config: Config, field: ConfigurableActionStructField,
             for k, v in value.items():
                 setattr(self, k, v)
 
+    @property
+    def _config(self) -> Optional[Config]:
+        return self._config_()
```

Comment:
My general worry about this pattern is that previously `_config` was never `None`, while after this change it is possible (may be improbable) that weakref returns `None` if original `Config` is gone. Could you make it return `Config` instead of `Optional[Config]` and raise an exception if weakref object is deleted? This applies to other packages where you use the same method, I'm not going to comment on it there.